jwst/assign_wcs/tests/test_nirspec.py

```diff
@@ -274,7 +274,7 @@ def test_msa_configuration_normal():
     # Test 1: Reasonably normal as well
     msa_meta_id = 12
     msaconfl = get_file_path('msa_configuration.fits')
-    slitlet_info = nirspec.get_open_msa_slits(msaconfl, msa_meta_id)
+    slitlet_info = nirspec.get_open_msa_slits(msaconfl, msa_meta_id, slit_y_range=[-.5, .5])
```

Comment:
Do you want to include `slit_y_range` in the call to `get_open_msa_slits` at line 291 as well?